src/Uno.UI.RuntimeTests/IntegrationTests/dxaml/controls/calendardatepicker/CalendarDatePickerIntegrationTests.cs

```diff
@@ -221,12 +220,12 @@ public async Task CanOpenFlyoutByKeyboard()
 			await TestServices.WindowHelper.WaitForIdle();
 
 			// press enter to open flyout
-			TestServices.KeyboardHelper.Enter();
+			TestServices.KeyboardHelper.Enter(FocusManager.GetFocusedElement() as UIElement);
```

Comment:
We shouldn't need to pass anything I think. Programmatic focus for CalendarDatePicker was done in line 216 already

Reply:
If this is required for some reason, then an UNO TODO comment would be nice

Reply:
I added the logic inside `PressKeySequence` instead.